src/System/TimedScopes/TimedScopeStack.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using Microsoft.Omex.System.Logging;
+using Microsoft.Omex.System.Validation;
+
+namespace Microsoft.Omex.System.TimedScopes
+{
+	/// <summary>
+	/// LinkedStack
+	/// </summary>
+	[Serializable]
+	public class TimedScopeStack
+	{
+		/// <summary>
+		/// Root item for all stacks
+		/// </summary>
+		public static TimedScopeStack Root { get; } = new TimedScopeStack();
```

Comment:
>static [](start = 9, length = 6)

Is it intended to be static?

Reply:
yes, it is used in TimedScopeStackManager.GetTimedScopeStack() method.